README.md

```diff
@@ -58,97 +61,84 @@ There are 1 API for prediction class
 ```ruby
 Datawow.prediction
 ```
----
-## Calling APIs
-Every classes there are 3 function that is `create`, `find_by` and `all`
-#### `create`
-```ruby
-Datawow.[class].create({data: "image URL", token: '_token'})
-```
-
-#### `find_by`
-```ruby
-Datawow.[class].find_by({id: "_image_id", token: '_token'})
-```
 
-#### `all`
+Those method, it's a shortcut for calling of these class
 ```ruby
-Datawow.[class].all({token: '_token'})
+Datawow::ImageClosedQuestion
+Datawow::PhotoTag
+Datawow::ImageChoice
+Datawow::ImageMessage
+Datawow::NanameueHuman
+Datawow::VideoClassification
+Datawow::TextClosedQuestion
+Datawow::TextCategory
+Datawow::TextConversation
+Datawow::TextJa
+Datawow::Prediction
 ```
 ---
-## Nanameue with Consensus
-##### There are 2 ways to use the library
-
-Firstly, __dynamic token__. if you have multiple project to work with, You could call library by using module name
+## Avalabile method for APIs
+Every classes there are 3 functions that is `create`, `find_by` and `all`
```

Comment:
Available methods in our APIs

There are 3 main functions for each class: `create`, `find_by`, and `all`